types/artifacts.d.ts

```diff
@@ -472,6 +474,15 @@ declare global {
         devicePixelRatio: number;
       }
 
+      export interface InspectorIssues {
+        MixedContent: Crdp.Audits.MixedContentIssueDetails[];
```

Comment:
Wait, this should be `mixedContent` lower case; capitalized first letter is (supposed to be) just for constructors.

Capitalized top-level artifact names was a confusing decision from a realllllly long time ago to get artifact names off of the gatherer `constructor` that we're stuck with now but we shouldn't extend it to their properties :)

(I'm kind of surprised eslint doesn't flag it, maybe we turned it off because of tests or something)